sections/persistenceOptimization.rst

```diff
@@ -1,10 +1,77 @@
 Persistence optimization
 ############################################
 
-hello world.
+Leftover signal in the following dark after a blast of illumination has been observed. It is called "Persistence". 
+Persistence has been observed in an early prototype E2V sensor as early as 2014 ([D2014]_). It was confirmed that the amplitude of the persistence decreased as the parallel swing voltage got smaller. This is consistent with the Residual Surface Image [J2001]_ -- the excessive charges are being stuck at the surface layer. The level of persistence is about 10--20 ADU, and the decaying time constant is about 30 sec [dmtn-276]_.
 
-- Trying new voltages
-- impact on persistence
-- impact on full-well
-- impact on other parameters
+During the EO testing in 2021, we also found the persistence made a streak toward the readout direction from the place where the bright spot located in a previous image. We call this trailing persistence.
```

Comment:
Maybe add one of Daniel's figures here showing the trail

Reply:
I'd like to limit materials we got from Run 7 so not to introduce any confusion. If we have images with trailling persistence from Run7 I'm happy to include it but if not (and I am sure NOT) I'd rather keep it referencing.